website/docs/index.html.markdown

```diff
@@ -29,6 +29,8 @@ Terraform supports a number of different methods for authenticating to Azure:
 
 We recommend using either a Service Principal or Managed Service Identity when running Terraform non-interactively (such as when running Terraform in a CI server) - and authenticating using the Azure CLI when running Terraform locally.
 
+->**Note:** The Service Principal or Managed Service Identity running Terraform should have permissions to register [Azure Resource Providers](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types). If the user running Terraform has insufficient permissions to register Resource Providers then we recommend setting the property [skip_provider_registration](#skip_provider_registration) in the provider block to prevent auto-registration.
```

Comment:
```suggestion
->**NOTE:** The Service Principal or Managed Service Identity running Terraform should have permissions to register [Azure Resource Providers](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types). If the user running Terraform has insufficient permissions to register Resource Providers then we recommend setting the property [skip_provider_registration](#skip_provider_registration) in the provider block to prevent auto-registration.
```

Not sure if there is a Hashi guideline/best practice for this, but thought I would mention this super minor documentation inconsistency here. Should **Note:** be, **NOTE:**, **Note:** or **note:**?

Reply:
AFAIK there's no guidelines and it's wildly inconsistent across all the docs - I personally go for **Note:** and in resources that I take care of I try be consistent with what's already in there. Uppercasing this to **NOTE:** would introduce an inconsistency to this particular page in the docs since all the other Notes are not uppercased.